ember-flight-icons/addon/components/flight-icon.js

```diff
@@ -64,4 +64,53 @@ export default class FlightIconComponent extends Component {
   get size() {
     return this.args.size ?? '16';
   }
+
+  /**
+   * Generates a unique ID for the title, if one exists
```

Comment:
```suggestion
   * Generates a unique ID for the title
```

Reply:
added in the commit that updates the test 👍